eppoclient/configurationrequestor.go

```diff
@@ -25,14 +25,7 @@ func newConfigurationRequestor(httpClient httpClient, configStore *configuration
 }
 
 func (ecr *configurationRequestor) GetConfiguration(experimentKey string) (flagConfiguration, error) {
-	if ecr.httpClient.isUnauthorized {
-		// should we panic here or return an error?
-		panic("Unauthorized: please check your SDK key")
-	}
```

Comment:
removing this behavior entirely

Reply:
This will conflict with https://github.com/Eppo-exp/golang-sdk/pull/50. I can rebase one PR on top of another, depending on which we merge first.

Reply:
I will wait for your PR to merge since it is larger in scope.